React/game/src/services/BookService.js

```diff
@@ -0,0 +1,5 @@
+import api from '../../config/api';
+
+export default {
+  getBookDetail: id => api.get('/books', { id })
+};
```

Comment:
why do we have a `BookService` file? It this isn't used anywhere, please delete it

Reply:
was created by an example of the card, I'm going to erase

Reply:
👍